### PR TITLE
feat(config): add Heatit Z-TRM6

### PR DESCRIPTION
**Pull request by geirra**
This adds config file for Heatit floor thermostat Z-TRM6

### CODE REVIEW

**Comment by geirra** [comment]
There seems to a issue with setpoints, should be resolved before merge.

**Comment by geirra** [comment]
as for the setpoints, it seems like we have to define a decimal point.
So i can set 24,5 but not 24,0 as that is translated to 24

Tried looking for a compat flag, but didn´t find any

**Comment by geirra** [comment]
Added compat override float, seems to work fine.

will do some more testing

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+			"minValue": 30,
+			"maxValue": 65535,
+			"defaultValue": 840,
+			"unit": "seconds",
+			"unsigned": true
+		},
+		{
+			"#": "21",
+			"label": "Action After Error",
+			"description": "Device will attempt to turn on again after timeout period",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]:
```suggestion
			"label": "Turn On Delay After Error",
```

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unit": "seconds",
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Turn off (Display error)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]:
```suggestion
					"label": "Stay off (Display error)",
```

Comment (AlCalzone) [reply packages/config/config/devices/0x019b/z-trm6.json]: together with the label change above this seems to make more sense?

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 2,
+			"unit": "10%",
+			"unsigned": true
+		},
+		{
+			"#": "26",
+			"label": "Thermostat State Update Interval",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]:
```suggestion
			"label": "Thermostat State Report Interval",
```

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 4]
```diff
@@ -0,0 +1,469 @@
...
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 43200,
+			"unsigned": true,
+			"unit": "seconds",
+			"options": [
+				{
+					"label": "On change",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]:
```suggestion
					"label": "Changes only",
```

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 5]
```diff
@@ -0,0 +1,469 @@
...
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "29",
+			"label": "Size of Load",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]:
```suggestion
			"label": "Load Power",
```
maybe?

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 6]
```diff
@@ -0,0 +1,469 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true,
+			"unit": "100W",
+			"options": [
+				{
+					"label": "Uses power metering value",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]:
```suggestion
					"label": "Use measured value",
```

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 7]
```diff
@@ -0,0 +1,469 @@
...
+			"options": [
+				{
+					"label": "Enable",
+					"value": 0
+				},
+				{
+					"label": "Disable",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]: We often call this Local Control. Would this make sense here too? In that case, the meaning of Enable and Disable would have to be switched.

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 8]
```diff
@@ -0,0 +1,469 @@
...
+		},
+		{
+			"#": "3",
+			"label": "Sensor Value(sen)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0,
+			"unsigned": true,
+			"unit": "KΩ",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]:
```suggestion
			"unit": "kΩ",
```

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 9]
```diff
@@ -0,0 +1,469 @@
...
+				{
+					"label": "33",
+					"value": 4
+				},
+				{
+					"label": "47",
+					"value": 5
+				},
+				{
+					"label": "6,8",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]: Is this supposed to be a comma between 6 and 8? Seems like a strange sequence of numbers in that case.

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 10]
```diff
@@ -0,0 +1,469 @@
...
+			"options": [
+				{
+					"label": "Uses power metering value",
+					"value": 0
+				}
+			]
+		}
+	],
+	"compat": {
+		"overrideFloatEncoding": {
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json:447]:
```suggestion
		// The device doesn't accept setpoints with automatic precision
		"overrideFloatEncoding": {
```

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 11]
```diff
@@ -0,0 +1,469 @@
+{
+	"manufacturer": "Heatit",
+	"manufacturerId": "0x019b",
+	"label": "Z-TRM6",
+	"description": "Floor thermostat",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]:
```suggestion
	"description": "Floor Thermostat",
```

File: packages/config/config/devices/0x019b/z-trm6.json [hunk 12]
```diff
@@ -0,0 +1,469 @@
...
+			]
+		},
+		{
+			"#": "16",
+			"label": "Active Display Brightness",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"unit": "10%",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-trm6.json]: All units with a number in them should have a space between the number and the actual unit
```suggestion
			"unit": "10 %",
```
please do the same for the other params too.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks - a few remarks.

File: packages/config/config/devices/0x019b/z-trm6.json (hunk 9 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x019b/z-trm6.json]: Ok seems to be intended. Weird... We should turn it into a decimal point for consistency though.
```suggestion
					"label": "6.8",
```

File: packages/config/config/devices/0x019b/z-trm6.json (hunk 2 above)
Comment (geirra) [reply packages/config/config/devices/0x019b/z-trm6.json]: agreed

File: packages/config/config/devices/0x019b/z-trm6.json (hunk 12 above)
Comment (geirra) [reply packages/config/config/devices/0x019b/z-trm6.json]: should be fixed for all params this now

File: packages/config/config/devices/0x019b/z-trm6.json (hunk 6 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x019b/z-trm6.json]: @geirra I think you missed this one

File: packages/config/config/devices/0x019b/z-trm6.json (hunk 7 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x019b/z-trm6.json]: @geirra I think you missed this one

Comment (geirra) [reply packages/config/config/devices/0x019b/z-trm6.json]: sorry, they where hidden in the filter, should be fixed now.

**Review by AlCalzone — APPROVED** [review]
Thanks. I did the last tweaks myself.

**Pull request merged**